test/e2e/pageobjects/OktaHome.js

```diff
@@ -3,14 +3,14 @@ class OktaLogin {
   get userMenu() { return $('[data-se="user-menu"] > a.link-button');}
   get signOutBtn() { return $('[data-se="user-menu"] a[data-se="logout-link"]'); }
   get popUpCloseButton() { return $('a[data-role="close-button"]'); }
+  get widgetFormTitle() { return $('.okta-form-title.o-form-head'); }
 
   async signOut() {
     await browser.waitUntil(async () => this.userMenu.then(el => el.isDisplayed()), 5000, 'wait for user menu');
     await this.userMenu.then(el => el.click());
     await browser.waitUntil(async () => this.signOutBtn.then(el => el.isDisplayed()), 5000, 'wait for signout btn');
-    const url = await browser.getUrl();
     await this.signOutBtn.then(el => el.click());
-    await browser.waitUntil(async () => browser.getUrl().then(cur => (cur !== url)), 5000, 'wait for url change');
+    await browser.waitUntil(async () => this.widgetFormTitle.then(el => el.isDisplayed()), 5000, 'wait for widget');
```

Comment:
In OIE org, the url after logout is the same as after login (yourorg.okta.com/app/userHome)
Hence this logic won't work to check if user is logged out. Instead, checking for the widget is a good check to ensure logout was successful